src/Commands/ImportDump.php

```diff
@@ -39,6 +40,9 @@ class ImportDump extends Command
      */
     protected $description = 'Imports a dump.';
 
+    protected const DOWNLOAD_REMOTE_DUMP       = 'Download remote dump';
```

Comment:
What's the advantage of having these defined in consts? 
I assume you just didn't want to type out `Download remote dump` twice in line 107?

Reply:
In case they get changed there is no way someone could mess up the duplicate string, besides it makes the string comparison less error-prone I guess.